### PR TITLE
chore(CHANGELOG): update to v4.8

### PR DESCRIPTION
**Pull request by philippfromme**
Related to https://github.com/camunda/camunda-modeler/issues/2221.

### CODE REVIEW

**Comment by barmac** [comment]
And this: https://github.com/camunda/camunda-modeler/issues/2214

**Comment by philippfromme** [comment]
@barmac Added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
...
+* `CHORE`: update to electron@12 ([#1926](https://github.com/camunda/camunda-modeler/issues/1926))
+* `CHORE`: update to zeebe-node@1.0.0 ([#2169](https://github.com/camunda/camunda-modeler/issues/2169))
+
+### BPMN
+
+* `FEAT`: automatically scroll canvas when creating new elements ([#1249](https://github.com/camunda/camunda-modeler/issues/1249))
+* `FIX`: copy root element references on replace ([#2185](https://github.com/camunda/camunda-modeler/issues/2185))
+* `FIX`: do not override existing documentation ([#1682](https://github.com/camunda/camunda-modeler/issues/1682))
+* `FIX`: Reconnect message flows when collapsing participant ([#1651](https://github.com/camunda/camunda-modeler/issues/1651))
+* `FIX`: Don't change namespace prefixes of the xml namespace ([#2214](https://github.com/camunda/camunda-modeler/issues/2214))
```
Comment (barmac) [inline CHANGELOG.md:28]: Let's not mix lowercase and uppercase

```suggestion
* `FIX`: reconnect message flows when collapsing participant ([#1651](https://github.com/camunda/camunda-modeler/issues/1651))
* `FIX`: don't change namespace prefixes of the xml namespace ([#2214](https://github.com/camunda/camunda-modeler/issues/2214))
```

Comment (philippfromme) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,28 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ## Unreleased
 
-___Note:__ Yet to be released changes appear here._
+___Note:__ Yet to be released changes appear here.
```
Comment (barmac) [inline CHANGELOG.md:7]:
```suggestion
___Note:__ Yet to be released changes appear here._
```

**Comment by barmac** [comment]
Oops, I changed the commit message accidentally (via squash and merge) :( Anyway, this is now merged and we can continue.

**Pull request merged**